Practice in php/calculator.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
Just a general comment on some of these file/folder names: I'd suggest not using spaces in any folders or files. Spaces aren't technically valid characters in a website address, so while it might work, it's not guaranteed to. Instead, try using underscores ("_") to replace the spaces.